src/views/v2/list/nav/next-disabled.php

```diff
@@ -11,12 +11,25 @@
  *
  * @var string $link The URL to the next page, if any, or an empty string.
  *
- * @version 4.9.4
+ * @version TBD
  *
  */
 ?>
 <li class="tribe-events-c-nav__list-item tribe-events-c-nav__list-item--next">
 	<button class="tribe-events-c-nav__next tribe-common-b2" disabled>
-		<?php echo esc_html( sprintf( __( 'Next %s', 'the-events-calendar' ), tribe_get_event_label_plural() ) ); ?>
+		<?php
+			echo sprintf(
```

Comment:
This looks OK, but I worry about translations with HTML. I'm also not the best person for best practices around this, @bordoni do you want to comment here?

Reply:
Should this also be in place for day view navigation?

Reply:
Yes, I wasn't sure either. While it's not recommended to have HTML in the translations, for these cases when you're translating a phrase, you don't have other options. If you split that into two words, then for some languages it doesn't work. And to do what they required on DQA (hiding the "Events" part on mobile) we don't have too many options besides that.

In the document, the request was only for list view. Maybe it's a matter of spacing and the length of the word "Events" vs "Day".

Reply:
Could we feasibly have 2 buttons, one with the short text and one with the long text and display one button instead the other depending on screen width?

I suppose doing so isn't super elegant, but it would allow us to remain translatable and flexible without markup.

Reply:
The behavior we want is:
```php
<?php
	$label = '<span class="tribe-events-c-nav__next-label-plural">' . tribe_get_event_label_plural() . '</span>'
	echo wp_kses(
		/* translators: %s: Event (plural or singular). */
		sprintf( __( 'Next %1$s' ), $label )
		[ 'span' => [ 'class' => [] ] ]
	);
?>
```

Reply:
Though wasn't Juan's point that separating out Next and Events to two separate strings it doesn't work for some languages.

Reply:
Though, I suppose as long as %s is used for the plural, then that'll always be a problem. You can ignore me :)

Reply:
I think Juan's point was that for some languages the order might not be the same as what we have in english, so we can't split the two words and only translate `Next` or `Previous`.

Reply:
In that case, @bordoni's recommendation is spot on.